ke/parser.py

```diff
@@ -11,8 +12,9 @@
 either          = matches? ( whitespace? '|' whitespace? matches? )+
 matches         = match ( whitespace? match )*
 match           = inner_literal / def / macro / braces
-macro           = '#' ( range_macro / token )
+macro           = '#' ( range_macro / multi_range_macro / token )
 range_macro     = range_endpoint '..' range_endpoint
+multi_range_macro = multi_range_endpoint '...' multi_range_endpoint
```

Comment:
lets make this two and not three dots, like regular ranges?

Reply:
Also, lets support negative numbers?

Reply:
You don't need to touch numrange.py to support negative numbers, just call it twice: `-(?:{negative})|{non-negative}` should do it